sources/install/package_cloud.sh

```diff
@@ -137,6 +137,11 @@ function install_azure_cli() {
     curl -sL https://aka.ms/InstallAzureCLIDeb -o /tmp/azure-cli-install.sh
     bash /tmp/azure-cli-install.sh
     rm /tmp/azure-cli-install.sh
+
+    # Move az from /opt directory
+    mv /opt/az /opt/tools/az
+    mv /usr/bin/az /opt/tools/bin/
```

Comment:
Because we are doing patching to change the install dir, I dont think moving the bin file is a good idea. If the package is updated with apt, some change might conflict. Not moving it should trigger a warning message to the user that try to update the package manually.